packages/dialtone-vue2/components/split_button/split_button.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+  <span
+    data-qa="dt-split-button"
+    class="d-split-btn"
+    :style="{ width: width }"
+  >
+    <dt-button
+      v-dt-tooltip="alphaTooltipText"
+      data-qa="dt-split-button-alpha"
+      :size="size"
+      :class="[`d-split-btn__alpha d-split-btn__alpha--${size}`]"
+      :active="alphaActive"
+      :aria-label="alphaAriaLabel"
+      :icon-position="alphaIconPosition"
+      :label-class="alphaLabelClass"
+      :loading="alphaLoading"
+      :assertive-on-focus="assertiveOnFocus"
+      :disabled="disabled"
+      :importance="importance"
+      :kind="kind"
+      @click="(e) => handleClick('alpha', e)"
+    >
+      <template #icon>
+        <!-- @slot Alpha (left) button icon slot -->
+        <slot name="alphaIcon" />
+      </template>
+      <!-- @slot Default content slot -->
+      <slot name="default" />
+    </dt-button>
+    <!-- @slot Omega (right) content slot -->
+    <slot name="omega">
+      <dt-dropdown v-if="!omegaId && $slots.dropdownList">
+        <template #anchor="{ attrs }">
+          <split-button-omega
+            v-bind="{ ...attrs, ...omegaButtonProps }"
+            @click="(e) => $emit('omega-clicked', e)"
+          >
+            <template #icon>
+              <!-- @slot Omega (right) button icon slot -->
+              <slot name="omegaIcon" />
+            </template>
+          </split-button-omega>
+        </template>
+        <template #list>
+          <!-- @slot Built-in dropdown contents -->
```

Comment:
bit confused just looking at the doc what should be passed in here. just list items? should it include the <ul> wrapper or not? May also want to mention omega slot will completely override this

Reply:
It's the dropdown's list slot, it's not documented properly on storybook either, but I'll make sure to mention this on the dialtone docs.

I'll add a small comment in the storybook slot, but it will be fully documented on dialtone.